src/app/common/error-messages.ts

```diff
@@ -4,10 +4,12 @@ export enum SendFormErrorMessages {
   SameAddress = 'Cannot send to yourself',
   AmountRequired = 'You must specify an amount',
   MustNotBeZero = 'Must be more than zero',
+  MustBeNumber = 'Amount of STX must be a number',
   DoesNotSupportDecimals = 'This token does not support decimal places',
   InsufficientBalance = 'Insufficient balance. Available:',
   MustSelectAsset = 'You must select a valid token to transfer',
   TooMuchPrecision = '{token} can only have {decimals} decimals',
   MemoExceedsLimit = 'Memo must be less than 34-bytes',
   AdjustedFeeExceedsBalance = 'The fee added now exceeds your current STX balance.',
+  CastToNumber = 'amount must be a `number` type, but the final value was: `NaN`',
```

Comment:
It looks like these error messages are typically started with a capitalized word, is there a reason this one would be different?